var/spack/repos/builtin/packages/py-awkward/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyAwkward(PythonPackage):
+    """Manipulate JSON-like data with NumPy-like idioms."""
+
+    git = "https://github.com/scikit-hep/awkward-1.0.git"
+    pypi = "awkward/awkward-1.1.2.tar.gz"
+    homepage = "https://awkward-array.org"
+
+    maintainers = ['vvolkl']
+
+    version('1.1.2', sha256='626e3a6a2a92dd67abc8692b1ebfa1b447b9594352d6ce8c86c37d7299dc4602')
+    version('0.3.1', sha256='7126d9feab8828b0b4f4c6dbc9e28c269a91e28eef4a6033d7ebb5db21f1dab3')
```

Comment:
These checksums need to be updated now, PyPI tarballs have different checksums than GitHub tarballs.

Reply:
Completely forgot about that, good catch!